### PR TITLE
fix deltann_compile.md

### PR DESCRIPTION
**Pull request by GaryGao99**
**Thank you for submitting a pull request! Please provide the following information for code review:**

# Pull Request Summary
Fix  tensorflow and deltann build.

# Test Plan
N/A

### CODE REVIEW

File: docs/installation/deltann_compile.md [hunk 1]
```diff
@@ -1,11 +1,140 @@
+
 # DELTA-NN compile
 
+Deltann support tensorflow， tensorflow lite，and tensorflow serving。
+
 ## Tensorflow C++
+Build tensorflow for Linux :
+
+1. Install deltann
```
Comment (zh794390558) [inline docs/installation/deltann_compile.md]: install under deltann docker.

File: docs/installation/deltann_compile.md [hunk 2]
```diff
@@ -1,11 +1,140 @@
...
+
+```
+cd tools/ && ./install/install-deltann.sh
+```
+
+2. Config tensorflow build
+```
+cd tools/tensorflow
+```
+Configure your system build by running the ./configure,
```
Comment (zh794390558) [inline docs/installation/deltann_compile.md]: give an example of gpu and andorid configure.

File: docs/installation/deltann_compile.md [hunk 3]
```diff
@@ -1,11 +1,140 @@
...
+bazel build -c opt --verbose_failures //tensorflow:libtensorflow_cc.so
+```
+**GPU support**
+
+For GPU support, set cuda=Y during configuration and specify the versions of CUDA and cuDNN.
+```
+bazel build -c opt --config=cuda --verbose_failures //tensorflow:libtensorflow_cc.so
+```
+**Tensoflow TensorRT support**
+
```
Comment (zh794390558) [inline docs/installation/deltann_compile.md]: add reference of install TF-TRT.

File: docs/installation/deltann_compile.md [hunk 4]
```diff
@@ -1,11 +1,140 @@
...
+cd tools/tensorflow
+```
+Configure your system build by running the ./configure,
+
+
+3. Build tensoflow library
+
+**CPU-only**
+```
+bazel build -c opt --verbose_failures //tensorflow:libtensorflow_cc.so
```
Comment (zh794390558) [inline docs/installation/deltann_compile.md]: support SIMD, mkl

File: docs/installation/deltann_compile.md [hunk 5]
```diff
@@ -1,11 +1,140 @@
...
+Edit "tensorflow/WORKSPACE". Add the following code to the end of the file.
+
+```
+android_ndk_repository(
+    name="androidndk",
+    path="/nfs/project/gaoyonghu/tools/android-ndk-r16b",
+    api_level=21
+)
+```
+2. Build tensoflow library
```
Comment (zh794390558) [inline docs/installation/deltann_compile.md]: how to tailor tflite library size.

Comment (GaryGao99) [reply docs/installation/deltann_compile.md]: I don't think there is an automatic way of doing this。You can edit tensorflow/lite/kernels/ register.cc and BUILD, delete some ops  that you don't require.

Comment (zh794390558) [reply docs/installation/deltann_compile.md]: can give a manual introduction for tailor size.

File: docs/installation/deltann_compile.md [hunk 6]
```diff
@@ -1,11 +1,241 @@
...
+
+3. Build deltann
+
+```
+cd delta/deltann && ./build.sh ios arm tflite
+```
+
+### Tailor tensorflow lite library
+
+There isn't an automatic way of doing this。You can edit tensorflow/lite/kernels/register.cc and tensorflow/lite/kernels/BUILD, delete some ops that you don't require.
```
Comment (zh794390558) [inline docs/installation/deltann_compile.md]: this。

Comment (zh794390558) [reply docs/installation/deltann_compile.md]: Eg：

**Pull request merged**